lib/api/actions/functions.php

```diff
@@ -489,17 +481,21 @@ function _beans_get_action( $id, $status ) {
 function _beans_set_action( $id, $action, $status, $overwrite = false ) {
 	$id = _beans_unique_action_id( $id );
 
-	// Get the action, if it's already registered.
-	$registered_action = _beans_get_action( $id, $status );
+	// If not overwriting, let's check if an action is already registered for that id.
+	if ( ! $overwrite ) {
+		// Get the action, if it's already registered.
```

Comment:
Aren't we just getting it here and checking if it is registered below?
So it would be `// Get the action.`
Or, if we don't want to be so verbose in the comments, use only one comment for getting, checking and returning?